daemon/list.go

```diff
@@ -516,6 +517,21 @@ func (daemon *Daemon) filterVolumes(vols []volume.Volume, filter filters.Args) (
 				continue
 			}
 		}
+		if filter.Include("label") {
+			v, err := daemon.volumes.Get(vol.Name())
```

Comment:
@cpuguy83 is there any reason why `List` doesn't return the labels? I'd rather be consistent and make both methods return the labels rather than using `Get` here, if it was possible.